strum/src/lib.rs

```diff
@@ -168,7 +168,7 @@ pub trait EnumProperty {
 
 /// A cheap reference-to-reference conversion. Used to convert a value to a
 /// reference value with `'static` lifetime within generic code.
-/// #[deprecated(since="0.13.0", note="please use `#[derive(IntoStaticStr)]` instead")]
+#[deprecated(since="0.13.0", note="please use `#[derive(IntoStaticStr)]` instead")]
```

Comment:
Sorry, I may not have been clear. Can you make this `since = "0.22.0"` instead of "0.13"